docs/pages/faq.mdx

```diff
@@ -127,3 +127,26 @@ are exceeding the usage limits of their license. The reporting library code is
 
 Reach out to `sales@goteleport.com` if you have questions about the commercial
 editions of Teleport.
+
+### Teleport Connect
+
+When you first start the app, Teleport Connect asks for permission to collect & send telemetry data.
+This includes tracking events such as:
+- Login
+- Starting an SSH, database, or kube session
+- File transfer during an SSH session
+- Managing access requests
+- Asking about job role
```

Comment:
@ravicious I added "asking about job role" too. Although it is a slightly different type of event (it is not collected in the background, but rather asked explicitly) I think it makes sense to have it on the list.